harmony-api/src/repos/albumRepo.ts

```diff
@@ -26,3 +28,23 @@ export const getAlbumTopicByPartialName = async (
     }
   })
 }
+
+export const getAlbumById = async (albumID?: number): Promise<Album> => {
+  const albumData = await prisma.album
+    .findUniqueOrThrow({
+      where: {
+        albumId: albumID,
+      },
+    })
+    .catch(() => {
+      throw new HttpError(`Album with id ${albumID} not found`, 404)
```

Comment:
This is not implemented, but going further I think it is a good idea to have error strings stored in a separate JSON file. Have a symbol ($) as placeholder for parameter in that error template and replace it accordingly.

For example:

import { allErrors }

generateError(allErrors.AlbumIdError, [{albumID}])

It has to be array to work for errors that require multiple custom parameters

Reply:
I'll do this in a different pull request, I wanna let Ikram do his thing